antsibull/galaxy.py

```diff
@@ -215,6 +226,13 @@ async def download(self, collection: str, version: t.Union[str, semver.Version],
             raise DownloadFailure(f'{release_url} failed to download correctly.'
                                   f' Expected checksum: {sha256sum}')
 
+        # Copy downloaded collection into cache
+        if self.collection_cache:
+            # TODO: PY3.8: We can use t.Final in __init__ instead of cast here.
+            cached_copy = os.path.join(t.cast(str, self.collection_cache),
+                                       release_info['artifact']['filename'])
+            shutil.copyfile(download_filename, cached_copy)
```

Comment:
We should probably do this for the ansible-base cache as well.

Reply:
True. That cache works differently though, since the ansible-base cache dir points to one checkout instead of a directory with checkouts (or tarballs). I can right now only imagine that this can work when `ansible_base_cache` points to an empty directory (copy the checkout / PyPi result) or non-existant directory (create it and copy). Removing the existing contents from that directory is nothing that we should do (f.ex. in case it happens to be our main ansible-base git checkout and in this run we need a PyPi version).

What do you think?

Reply:
Yeah, that's tricky isn't it... If it is a cache, then we should be able to blow it away.  But the way we're using it, it isn't a cache... more like a local install source (which we're currently using verbatim rather than trying to install from it).

Okay, I think separate PR is warranted for what we do with that... Maybe rename it to --ansible-base-install-source and change how we use it.  I'll open a ticket to track that.

Reply:
Here's the issue I've opened: https://github.com/ansible-community/antsibull/issues/105